randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui

```diff
@@ -143,7 +143,7 @@
            <item>
             <widget class="QCheckBox" name="surface_crumbles_check">
              <property name="text">
-              <string>Crumble Blocks in Surface - Cavern Cavity</string>
+              <string>Crumble Blocks in Surface East - Cavern Cavity</string>
```

Comment:
I thought we intentionally left out the "East" here?

Reply:
Likely not. Plus, that's not helpful if someone were to open the DB to find the room.